compiler/rustc_ast/src/ast.rs

```diff
@@ -1739,6 +1739,8 @@ pub struct MacroDef {
 #[derive(Clone, Encodable, Decodable, Debug, Copy, Hash, Eq, PartialEq)]
 #[derive(HashStable_Generic)]
 pub enum StrStyle {
+    /// A string from `include_str!` or `include_bytes!`.
```

Comment:
```suggestion
    /// A string from `include_str!` or `include_bytes!`.
```
`include_str!` will never result in `StrStyle::Included` as writter.